autoload/nvimdev.vim

```diff
@@ -46,6 +46,11 @@ function! nvimdev#init(path) abort
   let c_makers = []
   let s:ctags_file = s:path . '/tmp/tags'
 
+  for inc in s:include_paths
+	echomsg inc
+    let &path.=','.inc
```

Comment:
This should prefix `s:path/`.  I'm using absolute paths so things still work even if `:cd` is used later or auto-cd is disabled.